src/arch/x86_64/kernel/kmain.c

```diff
@@ -167,6 +172,16 @@ void kmain(uint64_t addr)
   alloc_init();
   print_ok();
 
+#ifdef ENABLE_FRAMEBUFFER
+  vbe_video_init(&entry->common);
+  video_clear(MAKE_RGB(0, 128, 0));
+  video_console_attach();
+
+#if ENABLE_FRAMEBUFFER_STATS
+  video_enable_debug_stats(true);
```

Comment:
Could we have this feature on the screen?

Reply:
yeah, tho we'd need to clear the screen as well

Reply:
might mean losing console access, or I can look into making ssfn print the console each frame

Reply:
ah well if that's complicated, let's forget about it

Reply:
I think I found a way around this, will update soon!

Reply:
Done!